aiosmtpd/docs/controller.rst

```diff
@@ -128,6 +148,64 @@ use to do some common tasks, and it's easy to write your own handler.  For a
 full overview of the methods that handler classes may implement, see the
 section on :ref:`handler hooks <hooks>`.
 
+Unix Socket-based Server
+------------------------
+
+The :class:`UnixSocketController` class creates a server listening to
+a Unix Socket (i.e., a special file that can act as a 'pipe' for interprocess
+communication).
+
+Usage is identical with the example described in the :ref:`tcpserver` section above,
+with some differences:
+
+**Rather than specifying a hostname:port to listen on, you specify the Socket's filepath:**
+
+.. doctest:: unix_socket
+    :skipif: in_win32
```

Comment:
The `:skipif:` there is why I revert from using pytest doctest and change to using `sphinx-build -b doctest` instead.